src/python/lib/ensembl/compara/config/mlss_species_set_adaptor.py

```diff
@@ -0,0 +1,52 @@
+"""
+See the NOTICE file distributed with this work for additional information
+regarding copyright ownership.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from typing import List
+from xml.etree import ElementTree
+import os
+
+def get_species_set_by_name(file: str, name: str) -> List[str]:
+    """Parses mlss_conf.xml file and returns a list of species for a specified species set name.
+
+     Args:
+        file: Path to the mlss_conf.xml.
+        name: Species set (collection) name.
+
+    Returns:
+        A list of species (genome names) for a specified species set name.
+
+    Raises:
+        FileNotFoundError: If the file is not found.
+        ElementTree.ParseError: If there are issues with parsing the file.
+        NameError: If the number of species sets with a specified name is not 1.
+    """
+    if not os.path.exists(file):
+         raise FileNotFoundError("mlss_conf file not found.")
+
+    tree = ElementTree.parse(file)
```

Comment:
Python's rule of asking for forgiveness instead of permission, i.e. let the `parse` method handle the error:
```suggestion
    tree = ElementTree.parse(mlss_conf_file)
```
(you may need to update `Raises` docstring and your unit tests with this change)